autotest/gcore/tiff_write.py

```diff
@@ -7331,6 +7331,36 @@ def test_tiff_write_jpeg_incompatible_of_paletted():
 
 
 ###############################################################################
+# Test blocksize overriding while creating (internal) overviews
+# on a newly created dataset
+
+def test_tiff_write_186_blocksize():
+    blockSizes = [64,256]
+    for blockSize in blockSizes:
+        src_ds = gdal.Open('../gdrivers/data/utm.tif')
+        fname = 'tmp/tiff_write_186_bs%d.tif' % blockSize
+
+        ds = gdal.GetDriverByName('GTiff').Create(fname, 1024, 1024, 1,
+                                                  options=['TILED=YES','COMPRESS=LZW',
+                                                      'BLOCKXSIZE=512', 'BLOCKYSIZE=512'])
+
+        data = src_ds.GetRasterBand(1).ReadRaster(0, 0, 512, 512, 1024, 1024)
+        ds.GetRasterBand(1).WriteRaster(0, 0, 1024, 1024, data)
+        opts = {'GDAL_TIFF_OVR_BLOCKSIZE':'%d'%blockSize}
+        if blockSizes == 256:
```

Comment:
@tbonfort `blockSize` not `blockSizes` yes?

Reply:
indeed! fixed. Thanks for the review